packages/core/contracts/common/implementation/Timer.sol

```diff
@@ -21,8 +21,7 @@ contract Timer {
     }
 
     /**
-     * @notice Gets the current time. Will return the last time set in `setCurrentTime` if running in test mode.
-     * Otherwise, it will return the block timestamp.
+     * @notice Gets the currentTime variable set in the Timer.
```

Comment:
OZ comment:
```
In Timer.sol the @notice docstring for the getCurrentTime function is misleading. It states, "Otherwise, it will return the block timestamp." However, the function is only capable of returning the stored value of currentTime and has no logic to conditionally return the block timestamp.

Reply:
Oh wow, great catch OZ.